lib/src/model/inheriting_container.dart

```diff
@@ -79,14 +79,14 @@ mixin Constructable on InheritingContainer {
 abstract class InheritingContainer extends Container
     with ExtensionTarget
     implements EnclosedElement {
-  late final DefinedElementType? supertype = () {
+  DefinedElementType? get supertype {
```

Comment:
We don't need caching these values?

Reply:
I ran some benchmarks that showed no increase in time-to-document, nor an increase in RSS (or at least, if there is, it's noise).

But if you're asking about returning a new instance at each call, dartdoc's "ModelElement"s are all cached in the package graph, so `modelBuilder.typeFrom` should return the same instance every time, when called with the same type and library, like on line 87.